site/cypress/e2e/Cypress-Gradeable/cyclic_grading_view.spec.js

```diff
@@ -0,0 +1,53 @@
+describe('Cyclic grading View Test', () => {
+    ['ta', 'instructor'].forEach((user) => {
+        it(`${user} cyclic grading view testing for Grading Homework`, () => {
+            cy.login(user);
+            cy.visit(['sample', 'gradeable', 'grading_homework', 'grading', 'status']);
+            cy.get('[data-testid="Grading Index"]').should('exist').click();
+            if (user === 'ta') {
+                cy.get('[data-testid="popup-window"]').should('exist');
+                cy.get('[data-testid="close-button"]').should('exist');
+                cy.get('[data-testid="close-hidden-button"]').should('exist');
+                cy.get('[data-testid="agree-popup-btn"]').should('exist').click();
+            }
+            cy.get('[data-testid="view-sections"]').click();
+            cy.get('[data-testid="grade-button"]').eq(12).click();
+            cy.get('[data-testid="student-info-btn"]').click();
+            cy.get('.rubric-title').should('contain', 'Joe Student');
+            cy.get('#prev-student-navlink').click();
+            cy.get('.rubric-title').should('contain', 'John Smith');
+            cy.get('#grading-setting-btn').click(); // change
+            cy.get('#general-setting-list').find('.ta-grading-setting-option').first().select('Prev/Next Ungraded Student');
```

Comment:
This would be better to create a test-id for the select element, instead of chaining find, and first.

Reply:
<img width="1353" alt="Screenshot 2024-05-24 at 5 11 36 PM" src="https://github.com/Submitty/Submitty/assets/119070053/d24d4c5f-a88c-4200-a8b2-91dc37f641eb">

removed chaining for this `cy.get('#general-setting-list').find('.ta-grading-setting-option').first().select('Prev/Next Ungraded Student');`
to `cy.get('[data-testid="ta_grading_setting_option"]').first().select('Prev/Next Ungraded Student');` 

but as you can clearly see in the image 3 component with same id or data-testid so for that reason i used the `.first`